mokapot/model.py

```diff
@@ -421,14 +446,15 @@ def __init__(
         override=False,
         subset_max_train=None,
         n_jobs=-1,
+        supervised_step=False,
```

Comment:
We should remove this. I think we should keep `PercolatorModel` as the most accurate replication of Percolator as possible - that means excluding the `supervised_init`

If someone really wanted to, they could still enable it like this:
```python
model = PercolatorModel()
model.supervised_init = True
```